### DISCUSSION TITLE
Account suspended with no reason, no support!

### DISCUSSION THREAD

**Discussion by ghost** [community/community#112646] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

My GitHub account [Support Ticket ID: 2670772] was suspended without notice or explanation, disrupting my projects and denying access to GitHub Copilot, for which I hold a yearly license! 😡 And it's been about a week!

This lack of communication and support is unacceptable! Please reinstate my account and restore Copilot access ASAP! 🙏

I've been using Github for many years and overall it's a good experience until NOW!!! I have to register a new account to post it here!!!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @RuiqiZhao1024, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
